dd-sdk-android-internal/src/main/java/com/datadog/android/internal/utils/ImageViewUtils.kt

```diff
@@ -80,11 +80,13 @@ object ImageViewUtils {
      * Resolves the [Drawable] content [Rect] using the given [ImageView] scale type.
      * @param imageView: the [ImageView].
      * @param drawable: the [Drawable].
+     * @param scaleType: optional custom [ImageView.ScaleType].
      * @return the resolved content [Rect].
      */
     fun resolveContentRectWithScaling(
         imageView: ImageView,
-        drawable: Drawable
+        drawable: Drawable,
+        scaleType: ImageView.ScaleType? = null
```

Comment:
perhaps rename to customScaleType to be the same as what we did with customResourceIdKey (naming to indicate that it's optional)?

Reply:
I agree, I have changed that.